components/telemetry-operator/README.md

```diff
@@ -2,83 +2,22 @@
 
 ## Overview
 
-The telemetry operator contains a logging controller that generates a Fluent Bit configuration from one or more LogPipeline custom resources. The controller ensures that all Fluent Bit Pods run the current configuration by deleting Pods after the configuration has changed. See all [CRD attributes](apis/telemetry/v1alpha1/logpipeline_types.go) and an [example](config/samples/telemetry_v1alpha1_logpipeline.yaml).
+The telemetry operator contains a logging controller that generates a Fluent Bit configuration from one or more LogPipeline and LogParser Custom Resources. The controller ensures that all Fluent Bit Pods run the current configuration by deleting Pods after the configuration has changed. See all [CRD attributes](apis/telemetry/v1alpha1/logpipeline_types.go) and some [examples](config/samples).
```

Comment:
```suggestion
The telemetry operator contains a logging controller that generates a Fluent Bit configuration from one or more LogPipeline and LogParser custom resources. The controller ensures that all Fluent Bit Pods run the current configuration by deleting Pods after the configuration has changed. See all [CRD attributes](apis/telemetry/v1alpha1/logpipeline_types.go) and some [examples](config/samples).
```
https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology#capitalization